pkg/network/node/egressip.go

```diff
@@ -153,18 +154,21 @@ func (eip *egressIPWatcher) ReleaseEgressIP(egressIP, nodeIP string) {
 	}
 }
 
-func (eip *egressIPWatcher) addEgressIP(nodeIP, egressIP string) {
+func (eip *egressIPWatcher) addEgressIP(nodeIP, egressIP, sdnIP string) {
 	eip.monitorNodesLock.Lock()
 	defer eip.monitorNodesLock.Unlock()
 
 	if eip.monitorNodes[nodeIP] != nil {
 		eip.monitorNodes[nodeIP].egressIPs.Insert(egressIP)
+		eip.monitorNodes[nodeIP].sdnIP = sdnIP
```

Comment:
This should not be needed; the sdn IP can't change while we have an entry for the node IP